docs/source/providers.rst

```diff
@@ -572,6 +572,9 @@ height, hash, timestamp, and other relevant data.
 the given address is mined on the blockchain. It provides information about the 
 tesseract.
 
+``PENDING_INTERACTIONS`` - This event is triggered when a new pending interaction
+occurs in the blockchain, providing a unique hash for the interaction.
```

Comment:
can be improved.

Reply:
25696762c8174ea9b4c5d3bab8ce0863d2d3cb5c